src/components/Txt/index.tsx

```diff
@@ -107,6 +155,12 @@ export interface InternalTxtProps extends React.HTMLAttributes<HTMLElement> {
 	align?: Align;
 	/** If true, replace the text not contained in the container with three dots */
 	truncate?: boolean;
+	/** Displays a copy icon which copies the given string to the user's clipboard onClick */
+	copy?: string;
+	/**  Setting to 'always' ensures that the copy icon is always visible. 'hover' ensures that the copy icon is only visible on hover. By default, this is set to 'hover'. */
+	showCopyMode?: 'hover' | 'always';
+	/** Displays Txt as inline-block and vertical aligns items at the bottom. Good for aligning text with icons. */
+	iconalign?: boolean;
```

Comment:
Why do we need this as a property? The Txt should use the appropriate display by default. If we want the user to be able to change it, we should have a property called `display` (it exists on both BoxProps and FlexProps). Alignment should be a separate property as well, likely and preferably from FlexProps or BoxProps, and it should have a good, sensible default value

Reply:
Agree with this. We could probably remove it without any replacement, unless I'm forgetting any strange cases